data/tlg0062/tlg012/tlg0062.tlg012.perseus-eng5.xml

```diff
@@ -0,0 +1,1630 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">A True History.</title>
+ <author>Lucian of Samosata</author>
+<editor role="translator">Emily James Smith</editor>
+ <funder>National Endowment for the Humanities</funder>
+ <principal>Gregory Crane</principal>
+ <respStmt>
+  <persName>Gregory Crane</persName>
+<resp>Digital conversion and editing</resp>
+ </respStmt>
+ </titleStmt>
+<publicationStmt>
+<authority>Perseus Digital Library</authority>
+<idno type="filename">tlg0062.tlg012.perseus-eng5.xml</idno>
+<availability>
+ <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+</availability>
+<date type="release">2023-08-01</date>
+<publisher>Harvard College Library</publisher>
+<pubPlace>United States</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+ <biblStruct>
+ <monogr>
+<title xml:lang="eng">Selections from Lucian</title>
+  <editor role="translator">Emily James Smith</editor>
+  <author ref="urn:cts:greekLit:tlg0062">Lucian of Samosata</author>
+  <imprint>
+   <publisher>Harper and Brothers</publisher>
+	<pubPlace>New York</pubPlace>
+	<date>1892</date>
+  </imprint>
+ </monogr>
+  <ref target="https://hdl.handle.net/2027/hvd.hw3j0n">The Hathi Trust</ref>
```

Comment:
We try to make the link to the page level of the work!

Reply:
added -- it should have been there but I was searching for archive.org rather than hdl.handle.